be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -546,6 +549,10 @@ Status HdfsOrcScanner::do_get_next(RuntimeState* runtime_state, ChunkPtr* chunk)
                         break;
                     }
                 }
+                if (chunk_size != 0) {
+                    ASSIGN_OR_RETURN(chunk_size, ExecNode::eval_conjuncts_into_filter(_scanner_params.conjunct_ctxs,
+                                                                                      ck.get(), &_chunk_filter));
```

Comment:
As conjunct_ctxs also contains contexts that exist in conjunct_ctxs_by_slot, so the single slot conjuncts will be executed repeatedly?

Reply:
No, conjunct_ctxs and conjunct_ctxs_by_slot are not overlapped. see `HiveDataSource::_decompose_conjunct_ctxs`

Reply:
```
struct HdfsScannerParams {
    // one file split (parition_id, file_path, file_length, offset, length, file_format)
    std::vector<const THdfsScanRange*> scan_ranges;

    // runtime bloom filter.
    const RuntimeFilterProbeCollector* runtime_filter_collector = nullptr;

    // should clone in scanner
    std::vector<ExprContext*> conjunct_ctxs;
    // conjunct group by slot
    // excluded from conjunct_ctxs.
    std::unordered_map<SlotId, std::vector<ExprContext*>> conjunct_ctxs_by_slot;
   ... 
```

Reply:
> No, conjunct_ctxs and conjunct_ctxs_by_slot are not overlapped. see `HiveDataSource::_decompose_conjunct_ctxs`

OK，I misunderstand it.

Reply:
> No, conjunct_ctxs and conjunct_ctxs_by_slot are not overlapped. see `HiveDataSource::_decompose_conjunct_ctxs`

OK